src/environments/environment.staging.ts

```diff
@@ -19,6 +19,7 @@ export const environment = {
   production: true,
   identityServerUrl: "https://staging-dex-identity.kn01.fhict.nl",
```

Comment:
```suggestion
  identityServerUrl: "https://identity.staging-dex.software",
```
We now have a new domain.

Reply:
are you sure its identity.staging-dex.software and not identity.staging.dex.software?

Reply:
You're right, corrected.